.circleci/config.yml

```diff
@@ -276,7 +276,8 @@ jobs:
           channel: "C7H40L71D" # dsde-qa-notify
   integration-tests-pr-staging:
     executor: puppeteer
-    parallelism: 4
+    parallelism: 3
```

Comment:
I'm not seeing a noticeable increase on length, but it would be worth having @mlilynolting try out these changes on her branch since the Azure runtime test is long.

Reply:
This is just a draft pr im experimenting with, ill have specific details on cost and time once I get some more data. In my experience so far, reducing the parallelism doesnt seem to extend the length of the tests by much, and if they are less flakey it should save credits/time overall.